protocol-units/execution/opt-executor/src/executor/execution.rs

```diff
@@ -65,9 +64,10 @@ impl Executor {
 		})
 		.await??;
 
-		info!("Block execution compute the following state: {:?}", state_compute);
+		warn!("Block execution compute the following state: {:?}", state_compute);
```

Comment:
This is a normal outcome, why use the WARN level?
I'd rather bump this down to debug to eliminate the spam from dumping `state_compute`.